Filters/exceptions.txt

```diff
@@ -3,6 +3,9 @@
 ! `*-dnsotls-ds.metric.gstatic.com` (for instance, `a5a6380f-dnsotls-ds.metric.gstatic.com`).
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/81939
+@@||support-beacon.nakanohito.jp^|
```

Comment:
Better exclude `||nakanohito.jp^` instead of unblocking(blocking rule will be removed from filter, and if user wants, he can block without addition modifiers).

Reply:
`nakanohito.jp` has various tracking subdomains (in [OISD hosts full](https://oisd.nl), 116 domains listed), but a few domains already included in AdGuard Tracking Protection cover most of real cases. OK, I agree with you.

Reply:
> has various tracking subdomains

I see. Now **I** agree with you:) `@@` are good there.